docs/manual/rules/axis_filter_rule.md

```diff
@@ -5,11 +5,10 @@ FORMAT
 - axis_filter_rule:
       name: <name_of_rule_in_text>
       description: <description_of_rule_in_text>
-      layer_number: <integer_nr_of_layer>
-	  axis_name: <name_of_axis_applied>
+      axis_name: <name_of_axis_applied>
       layer_number: <integer_nr_of_layer_in_axis_applied>
       input_variable: <one_3D_input_variable_name>
-      output_variable: <one_output_variable_name>
+      output_variable: <one_output_variable_name> 
```

Comment:
why add a trailing space?

Reply:
if you open the file, it is a sea of trailing spaces :) updated it. No more trail of spaces.